### PR TITLE
Add MAX_SAFE_INTEGER polyfill

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: polyfills/Number/MAX_SAFE_INTEGER/config.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+		"firefox": "<31",
+		"firefox_mob":"<32",
+		"chrome":"<34",
+		"android": "<5",
+		"safari": "<9",
+		"ios_saf": "<9",
+		"opera": "*",
+		"op_mini": "*",
+		"op_mob": "*",
+		"bb": "*"
```
Comment (ruiaraujo) [inline polyfills/Number/MAX_SAFE_INTEGER/config.json:17]: Missing samsung browser here.

Comment (JakeChampion) [reply polyfills/Number/MAX_SAFE_INTEGER/config.json:17]: We don't know the support for Samsung browser -- @AdaRoseEdwards could you help here?

Comment (AdaRoseCannon) [reply polyfills/Number/MAX_SAFE_INTEGER/config.json:17]: We have it
image

Comment (JakeChampion) [reply polyfills/Number/MAX_SAFE_INTEGER/config.json:17]: Then there is no need to add it into this config file :-) Thanks @AdaRoseEdwards!

Comment (ruiaraujo) [reply polyfills/Number/MAX_SAFE_INTEGER/config.json:17]: @AdaRoseEdwards what version of the Samsung browser is that?

**Review by rowanmanning — APPROVED** [review]
Looks good 👍

File: polyfills/Number/MAX_SAFE_INTEGER/config.json (hunk 1 above)
Comment (JakeChampion) [reply polyfills/Number/MAX_SAFE_INTEGER/config.json:17]: We support Samsung 4 & above -- https://cdn.polyfill.io/v2/docs/contributing/authoring-polyfills#browsers

Comment (AdaRoseCannon) [reply polyfills/Number/MAX_SAFE_INTEGER/config.json:17]: @ruiaraujo 4.0, I think 4.2 has only been released on note7.

**Pull request merged**